packages/checkout/widgets-lib/src/widgets/sale/components/CreditCardWarningDrawer.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Body, Box, Button, Drawer, Heading,
+} from '@biom3/react';
+import { SalePaymentTypes } from '@imtbl/checkout-sdk';
+import { CreditCardWarningHero } from 'components/Hero/CreditCardWarningHero';
+import { useTranslation } from 'react-i18next';
+
+type CreditCardWarningDrawerProps = {
+  visible: boolean;
+  setShowCreditCardWarning: (show: boolean) => void;
+  setPaymentMethod: (type: SalePaymentTypes) => void;
+};
+
+export function CreditCardWarningDrawer({
+  visible,
+  setShowCreditCardWarning,
+  setPaymentMethod,
+}: CreditCardWarningDrawerProps) {
+  const { t } = useTranslation();
+
+  return (
+    <Drawer
+      size="threeQuarter"
+      visible={visible}
+      showHeaderBar={false}
```

Comment:
@mimi-imtbl Minor thing - should this be showing the close icon? Noticed it's in the Jira ticket, but can't find it in Figma to confirm.

Reply:
Was planning to bring this up with Cass later, but if we show the header bar (that includes the close button), it will take the horizontal space and push the hero images down. The drawer is also closeable from clicking anywhere outside of it.

Reply:
CTL: Cass agreed to change the drawer to `full` to accommodate the header space while showing the close button and header